config/build.ts

```diff
@@ -464,23 +458,55 @@ async function generatePlaylists(dir, processedSnippets: Dictionary<SnippetProce
         */
         const groupNumberRegex = /^(\d+\s)?(\w.*)$/;
 
-        items.forEach(item => {
+        let modifiedItems = cloneDeep(items);
```

Comment:
Instead of cloning and deleting items that we do not want, I think it may be cleaner to do a map on the array instead and select the items we want. Something like:
playlistItem = items.map({name, description, etc} => {
return {name, description, etc};
});